### PR TITLE
Add new story from 1927

### PR DESCRIPTION
**Pull request by b-t-k**

### CODE REVIEW

File: src/epub/text/those-in-peril-on-the-tee.xhtml [hunk 1]
```diff
@@ -248,12 +248,12 @@
...
 			<p>“Agnes,” he said, huskily, “there is something I want to say to you. Oh, Agnes, have you not guessed⁠—”</p>
 			<p>“One moment,” said Agnes Flack. “If you’re trying to propose to me, sign off. There is nothing doing. The idea is all wet.”</p>
 			<p>“All wet?”</p>
-			<p>“All absolutely wet. I admit that there was a time when I toyed with the idea of marrying a man with brains, but there are limits. I wouldn’t marry a man who played golf as badly as you do if he were the last man in the world. Sid-nee!” she roared, turning and cupping her mouth with her hands; and a nervous golfer down by the lake-hole leaped three feet and got his mashie entangled between his legs.</p>
+			<p>“All absolutely wet. I admit that there was a time when I toyed with the idea of marrying a man with brains, but there are limits. I wouldn’t marry a man who played golf as badly as you do if he were the last man in the world. Sid-née!” she roared, turning and cupping her mouth with her hands; and a nervous golfer down by the lake-hole leaped three feet and got his mashie entangled between his legs.</p>
```
Comment (acabal) [inline src/epub/text/those-in-peril-on-the-tee.xhtml]: Wrong change

File: src/epub/text/colophon.xhtml [hunk 2]
```diff
@@ -23,7 +23,7 @@
 			for<br/>
 			<a href="https://standardebooks.org/ebooks/p-g-wodehouse/golf-stories#transcriptions">Project Gutenberg</a><br/>
 			and on digital scans from the<br/>
-			<a href="https://standardebooks.org/ebooks/p-g-wodehouse/golf-stories#page-scans">Internet Archive</a>.</p>
```
Comment (acabal) [inline src/epub/text/colophon.xhtml:26]: If there is more than one source, we say "various sources" https://standardebooks.org/manual/1.7.0/single-page#6.6.6.1.3.2.2

File: src/epub/text/imprint.xhtml [hunk 3]
```diff
@@ -12,7 +12,7 @@
 				<img alt="The Standard Ebooks logo." src="../images/logo.svg" epub:type="z3998:publisher-logo se:image.color-depth.black-on-transparent"/>
 			</header>
 			<p>This ebook is the product of many hours of hard work by volunteers for <a href="https://standardebooks.org">Standard Ebooks</a>, and builds on the hard work of other literature lovers made possible by the public domain.</p>
-			<p>This particular ebook is based on transcriptions from <a href="https://standardebooks.org/ebooks/p-g-wodehouse/golf-stories#transcriptions">Project Gutenberg</a> and on digital scans from the <a href="https://standardebooks.org/ebooks/p-g-wodehouse/golf-stories#page-scans">Internet Archive</a>.</p>
```
Comment (acabal) [inline src/epub/text/imprint.xhtml:15]: Same here https://standardebooks.org/manual/1.7.0/single-page#6.4.4.3.2

**Comment by acabal** [comment]
Great! A few minor issues in the review.

**Review by acabal — CHANGES_REQUESTED** [review]

File: src/epub/css/core.css [hunk 4]
```diff
@@ -99,12 +99,6 @@ i > em{
 	font-style: normal;
 }
 
-ol,
-ul{
```
Comment (acabal) [inline src/epub/css/core.css]: These have to stay as part of the new template

**Review by acabal — CHANGES_REQUESTED** [review]

File: src/epub/text/those-in-peril-on-the-tee.xhtml (hunk 1 above)
Comment (b-t-k) [reply src/epub/text/those-in-peril-on-the-tee.xhtml]: ? She is yelling Sidney in an elongated fashion... no need for an accent.

File: src/epub/text/colophon.xhtml (hunk 2 above)
Comment (b-t-k) [reply src/epub/text/colophon.xhtml:26]: Missed that in the manual. Time to re-read the whole thing...

**Comment by b-t-k** [comment]
Done!

**Comment by acabal** [comment]
Thanks!

**Pull request merged**